packages/node/package.json

```diff
@@ -39,6 +39,7 @@
   "dependencies": {
     "@progress/jsdo-core": "^6.0.0",
     "base-64": "^0.1.0",
+    "node-localstorage": "^1.3.1",
```

Comment:
We removed the requirement the node-localstorage module. It is not expected to be in the latest package.json.